nests.json

```diff
@@ -346,6 +346,17 @@
       "status": "stable",
       "category": "scientific",
       "readme": "https://raw.githubusercontent.com/Libraries4U/Box2D/main/README.md"
-    }
+    },
+    {
+      "name": "Skylark Uploaders",
+      "packages": [
+        "SkylarkUploaders"
+      ],
+      "description": "Skylark uploaders is a website allowing to upload / download files. It require Skylark to be compiled.",
```

Comment:
Could it be used as extension or it is single product? I think maybe we should have categories webapp instead? @ismail-yilmaz what do you think?